src/Miunie.Discord/CommandModules/RemoteRepositoryCommand.cs

```diff
@@ -33,7 +34,7 @@ public RemoteRepositoryCommand(RemoteRepositoryService remoteRepoService, Entity
         }
 
         [Command("repo")]
-        [Summary("Shows the official remote repository hosting the code of this bot")]
```

Comment:
(about the `.resx`)
Especially since we already had a summary written out in a plain string. I'm pretty sure we discussed it, but decided against `.resx`.